client/my-sites/stats/wordads/index.jsx

```diff
@@ -136,6 +137,10 @@ class WordAds extends Component {
 		);
 	};
 
+	isProduction() {
```

Comment:
Could we use feature flags instead of a URL check? It could be done in a similar way to [this](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/stats/stats-insights/index.jsx#L31).

Reply:
I purposefully avoided a feature flag in this case as this is a very localized change. If I didn't anticipate a follow up PR for the icons, I wouldn't need a check at all. Now that I've added in the icons, I think we can just ship this as production-ready though.

I'll make those changes first, then revisit this.

Reply:
This check has been removed now!